xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -313,12 +319,13 @@ namespace PVR
    const std::string GetGenresLabel() const;
 
   private:
-    CDateTime    m_recordingTime; /*!< start time of the recording */
+    CDateTime    m_recordingTime;     /*!< start time of the recording */
     bool         m_bGotMetaData;
-    bool         m_bIsDeleted;    /*!< set if entry is a deleted recording which can be undelete */
-    unsigned int m_iEpgEventId;   /*!< epg broadcast id associated with this recording */
-    int          m_iChannelUid;   /*!< channel uid associated with this recording */
-    bool         m_bRadio;        /*!< radio or tv recording */
+    bool         m_bIsDeleted;        /*!< set if entry is a deleted recording which can be undelete */
+    unsigned int m_iEpgEventId;       /*!< epg broadcast id associated with this recording */
+    int          m_iChannelUid;       /*!< channel uid associated with this recording */
+    int          m_iRecordingState;   /*!< recording state associated with this recording */
+    bool         m_bRadio;            /*!< radio or tv recording */
```

Comment:
Please do not touch the blanks to "beautify" the code. We don't want that horizontal alignment. See our coding guidelines.